src/main/java/org/cryptomator/cryptofs/CryptoFileSystemProperties.java

```diff
@@ -80,6 +81,15 @@ public class CryptoFileSystemProperties extends AbstractMap<String, Object> {
 
 	static final String DEFAULT_MASTERKEY_FILENAME = "masterkey.cryptomator";
 
+	/**
+	 * Key identifying the function to call for notifications.
+	 *
+	 * @since 2.9.0
+	 */
+	public static final String PROPERTY_NOTIFY_METHOD = "notificationConsumer";
+
+	static final Consumer<FilesystemEvent> DEFAULT_NOTIFY_METHOD = (FilesystemEvent e) -> {};

```

Comment:
To keep naming coherent, I'd suggest:

```suggestion
	public static final String PROPERTY_EVENT_CONSUMER = "fsEventConsumer";
	
	static final Consumer<FilesystemEvent> DEFAULT_EVENT_CONSUMER = (FilesystemEvent e) -> {};

```

Reply:
Renamed in https://github.com/cryptomator/cryptofs/pull/277/commits/3691c6bd5a17901ffee920e83e3bae7e3ce70d06